cmake/WtFindZlib.txt

```diff
@@ -6,6 +6,18 @@
 # Taking into account:
 # - ZLIB_PREFIX
 
+IF(HUNTER_ENABLED)
+  hunter_add_package(ZLIB)
+  find_package(ZLIB)
```

Comment:
Should be `find_package(ZLIB CONFIG REQUIRED)`: https://docs.hunter.sh/en/latest/packages/pkg/ZLIB.html